UserAgent.xcodeproj/xcshareddata/xcschemes/Cliqz.xcscheme

```diff
@@ -161,6 +161,15 @@
                <Test
                   Identifier = "DesktopModeTestsIphone">
                </Test>
+               <Test
```

Comment:
How about we remove those tests completely? 

On the other hand would nice to have at least a basic coverage for Dark Mode. Can we adopt any those tests to test iOS 13 notification?

Reply:
I'm all for removing the tests

Reply:
The dark mode on/off state we are getting when app is coming active. There is no iOS native notification for dark mode. So I need to investigate test mechanist to understand does it allow us to change system settings from test environment, etc.. @chrmod Do we need to do this investigation?